stable_baselines3/common/vec_env/vec_normalize.py

```diff
@@ -179,16 +179,20 @@ def save(self, save_path):
 
     def save_running_average(self, path):
         """
-        :param path: (str) path to log dir
+        :param path: (str) path to pickle file where to store statistics
```

Comment:
I thought those two were deprecated and `.save()` `.load()` were prefered?
(so we can remove them)

Reply:
the reason they are still there is maybe that I was trying to keep SB2 and SB3 in sync